Describe linked issues can use trailer syntax (#36607)
github/docs: content/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue.md

Issue in a different repository | KEYWORD OWNER/REPOSITORY#ISSUE-NUMBER | `Fixes octo-org/octo-repo#100`
Multiple issues | Use full syntax for each issue | `Resolves #10, resolves #123, resolves octo-org/octo-repo#100`

> The keywords can be capitalized and followed by colons like the form of Git trailer convention, e.g. `Closes: #10`.

Reviewer: ```suggestion
The keywords can be followed by colons or in uppercase. For example: `Closes: #10`, `CLOSES #10`, or `CLOSES: #10`.
```
